app/javascript/angular/code/services/_draw_session.js

```diff
@@ -8,7 +8,7 @@ export const drawSession = (sensors, map, heat, empty) => {
 
   DrawSession.prototype = {
     drawMobileSession: function(session, drawSessionStartingMarker) {
-      if (!session || !sensors.anySelected()) {
```

Comment:
I think this is a left over from the times when you could select all sensors - not needed any more